package_managers/apt_key.bzl

```diff
@@ -0,0 +1,51 @@
+#Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rule for configuring apt GPG keys"""
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
+load("//util:run.bzl", "container_run_and_extract")
+
+def add_apt_key(name, key, image):
+    initial_image = "//package_managers:apt_key_image"
```

Comment:
Should make this "initial_image" to gcr.io/gcp-runtimes/apt_key:latest? via container_pull rule?
https://github.com/bazelbuild/rules_docker/blob/5f58b022f191722d5771636478691ed13346b3ee/container/pull.bzl#L82

Reply:
I was trying to avoid actually publishing this image, but I think it might be necessary for use in other repos.